### DISCUSSION TITLE
options explanation needed

### DISCUSSION THREAD

**Discussion by kingnixon** [tosdr/browser-extensions#63] [Q&A] [answered]
I opened my browser today and tosdr popped up a page to set the extension's new options.  But there is no explanation anywhere I can find of what they are or do. Curator mode? Privacy shields? Unspecified notifications? These options are completely opaque and meaningless.

**Comment by JustinBack** [comment, marked as answer]
While I agree that some options do not have an explanation, "Send Notifications" should be clear :-)


Curator Mode -> Something specific for our Tos;DR Curators, enabling shortcuts to Phoenix

![](https://9888-983528b48d7266437-14a87dc0-4ce4-11eb-b59c-5404a6a07802.imp.jbcdn.net/captures/hSVCOCg68z.png)

Privacy Shields -> Are your small badges to embed in a website, these can be turned off using that option to reduce cluttering and web requests

![](https://9888-983528b48d7266437-14a87dc0-4ce4-11eb-b59c-5404a6a07802.imp.jbcdn.net/captures/rtPpj6bm6P.png)

**Reply by kingnixon** [reply]
thanks! since there have been notifications for I think as long as I've been using the app, i wasn't sure if that was asking about something new or what's already there

**Reply by JustinBack** [reply]
Oh yeah, in general it was asked a lot to have the ability to disable the notifications as they can be annoying.
